src/http-spec.ts

```diff
@@ -1,6 +1,6 @@
+import { JSONSchema4, JSONSchema6, JSONSchema7 } from 'json-schema';
```

Comment:
Do we need to add a dependency or peerdep on `json-schema@7`? Running into an issue adding `@stoplight/types@9` to `api-gen3

![image](https://user-images.githubusercontent.com/7423098/60555429-5bf83780-9d02-11e9-83a7-b179a47f302e.png)

Reply:
Yeah - `@types/json-schema` is in devDependencies right now - it should be in dependencies.

Reply:
Sounds good, I'll move it over 👍

Reply:
Ah gosh, this was my bad!

Reply:
No worries! I think I was the first to run into it but all resolved now 🎉